colab_logica.py

```diff
@@ -309,17 +317,21 @@ def Logica(line, cell, run_query):
                 color.Warn(predicate + '_sql'))
 
   with bar.output_to(logs_idx):
+    print(sql)
```

Comment:
Can you please remove the print? :-)

Reply:
That's for debugging purpose. Will remove it once we finish passing all tests